app/AbstractRequest.php

```diff
@@ -450,7 +454,7 @@ public function cacheKey(): string
                 self::class,
                 $this->getURL(),
                 $this->encodeBody(),
-                config('api-request.cache_key_seed', 'v2022.4.12.0'),
+                config('api-request.cache_key_seed', 'v2024.8.6'),
```

Comment:
This is what we designed the seed for, the whole format of the cached object is changing, so the seed changes to invalidate what was cached already.